Classes/MenugScene.cpp

```diff
@@ -129,7 +131,7 @@ void Menug::setupButtons()
 
 void Menug::setupVersionLabel()
 {
-    Label* label1 = Label::createWithTTF("Version 0.1", "fonts/AndikaLowerCase-Regular_5dp.ttf", 20);
+    Label* label1 = Label::createWithTTF(LanguageManager::getString("language").c_str(), "fonts/AndikaLowerCase-Regular_5dp.ttf", 10);
```

Comment:
I think is better to show what language is running in the game than a version number.